ajax/dropdown_values.php

```diff
@@ -3,13 +3,14 @@
 
 Session::checkRight("entity", UPDATE);
 
-if(class_exists($_REQUEST['dropdown_itemtype'])) {
+if($_REQUEST['dropdown_itemtype'] == '0' || !class_exists($_REQUEST['dropdown_itemtype'])) {
+   $options = array(
+         'display_emptychoice'   => true,
+   );
+   Dropdown::showFromArray("dropdown_default_value", array(), $options);
```

Comment:
Indeed, I checked and this is short enough to not hit ehe line length limit. 

I prever array(...) instead of [...] for now because the plugin is still compatible with old GLPI versions.